data/events.json

```diff
@@ -0,0 +1,3 @@
+{"ref":"events.json",
```

Comment:
Do you mind pretty-formatting the test JSON data? These files could be good places to view the shape of the data as reference during development. :)

Reply:
Done! :D